src/module-elasticsuite-tracker/Api/Customer/ReconciliationServiceInterface.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile Searchandising Suite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteTracker
+ * @author    Romain Ruaud <romain.ruaud@smile.fr>
+ * @copyright 2018 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+namespace Smile\ElasticsuiteTracker\Api\Customer;
+
+/**
+ * Customer Reconciliation : used to handle correlation between customer_id and session_id/visitor_id
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteTracker
+ * @author   Romain Ruaud <romain.ruaud@smile.fr>
+ */
+interface ReconciliationServiceInterface
```

Comment:
Not sure if I want to name this ReconciliationServiceInterface.
I think that CustomerTrackingServiceInterface would be a better name.